recipes/popoolation2/001-perl-path.patch

```diff
@@ -0,0 +1,126 @@
+diff -u -r ./cmh-test.pl ../popoolation2_1201/cmh-test.pl
+--- ./cmh-test.pl	2011-09-20 13:21:57.000000000 -0500
++++ ../popoolation2_1201/cmh-test.pl	2023-05-22 17:46:15.143501024 -0500
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env perl
++#!/opt/anaconda1anaconda2anaconda3/bin/perl
```

Comment:
Why is this needed ?
I haven't seen other recipes doing it.

Reply:
This is exactly as the recipe appears on the HCC conda channel. I haven't modified it, nor am I the original author, so I'm not sure

Reply:
It's probably not needed